### PR TITLE
[WIP] Install activeadmin 4294

### PR DESCRIPTION
**Pull request by dmitrytrager**
Accroding to: https://redmine.smartdata.solutions/issues/4294

This PR does not tune up specific view in admin panel. It only helps to install active admin and provides an ability to see users list like this:
users

### CODE REVIEW

File: config/initializers/kaminari.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# config/initializers/kaminari.rb
+Kaminari.configure do |config|
```
Comment (dmitrytrager) [inline config/initializers/kaminari.rb]: need this b/c of using `will_paginate`

File: app/models/user.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable, :recoverable, :rememberable, :trackable, :validatable,
          authentication_keys: { username: false, email: true }
 
-  validates :username, uniqueness: true, allow_blank: true,
+  validates :username, uniqueness: true, allow_blank: false,
```
Comment (dmitrytrager) [inline app/models/user.rb]: DB already has NULL constraint. There is no point to allow NULL here.

Comment (t-harps) [inline app/models/user.rb]: We allow users to have blank usernames though, it gets saved as an empty string i believe. The usernames are currently only to make the url's look nicer, so we left them optional.

Comment (dmitrytrager) [inline app/models/user.rb]: thanks @t-harps 
I've already rolled this change back

**Comment by dmitrytrager** [comment]
@t-harps @GarySiu guys, need help with CI, please

**Pull request merged**